### ISSUE TITLE
Bot stops handling commands after IllegalAccessError

### ISSUE THREAD

**Issue by MyMindWorld** [kotlin-telegram-bot/kotlin-telegram-bot#182]
**Expected behavior**
When any exception is thrown inside handler bot shoud handle comands afterwards

**Actual behavior**
When Illegal AccessError is thrown - bot stops responding

**To Reproduce**
```kotlin
dispatch {

    text("br") {
//      throw IllegalAccessError("Illegal") // Breaks
        throw java.lang.IllegalStateException("Illegal") // Doesn't Breaks
    }
    text("ping") {
        bot.sendMessage(
            chatId = ChatId.fromId(message.from!!.id), text = "Pong"
        )
    }
}
```

**Possible solutions**
Wrap handler to catch generic exception

**Comment by vjgarciag96** [comment]
Hi @MyMindWorld!

The problem with `IllegalAccessError` is that it's not an `Exception`, it's an `Error` and we are not catching them. 

I don't have a strong opinion about whether we should be also catching errors and not. It would be convenient to keep the bot running if an error is thrown, but I don't know if that could lead to a weird state in which the program caught an error but is not able to continue running properly.

I've been looking to other libraries and it seems it's common to catch them all, so I guess it make sense to do the same here. I'll do the change and will monitor if everything is fine with it.

**Comment by vjgarciag96** [comment]
It should be fixed in version 6.0.4 - let me know if you find any problem. 

And thanks for the report!

**Comment by MyMindWorld** [comment]
Many thanks for quick response and explanation!
